new_samples/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"github.com/uber-common/cadence-samples/new_samples/worker"
+	"log"
+	"net/http"
+)
+
+func main() {
+	worker.StartWorker()
+	log.Println("Cadence worker service started at port: 3000")
+	err := http.ListenAndServe(":3000", nil)
+	if err != nil {
+		log.Fatalf("cadence server bootstrap failed: %v", err)
+	}
```

Comment:
I don't think this is correct about what it's doing, or a good fit for the intent.

I _believe_ this is using an http server as a "ctrl-c to stop" tool?  there are simpler ways to achieve that if so (and MUCH less likely to cause issues, e.g. you can't run more than one of these).

Reply:
Yeah you are right. Instead of using http server, I changed to use fx. Also tested on my local I'm able to start multiple fx process.